sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1270,6 +1283,10 @@ ContextImplPtr MemCpyCommandHost::getContext() const {
   return detail::getSyclObjImpl(Queue->get_context());
 }
 
+QueueImplPtr MemCpyCommandHost::getWorkerQueue() const {
```

Comment:
Can reference to QueueImplPtr be returned here to avoid unnecessary copies?

Reply:
Sure, this was done this way mostly to mirror the existing `getContext`/`getQueue` methods. Do you mind if all of these are modified in a separate patch?

Reply:
I do not mind.